mlflow/tensorflow.py

```diff
@@ -912,7 +912,15 @@ def train(original, self, *args, **kwargs):
             try_mlflow_log(mlflow.log_param, "max_steps", kwargs["max_steps"])
 
         result = original(self, *args, **kwargs)
-
+        # Logs Tensorboard to artifacts.
```

Comment:
Minor suggestion for inline comment:

```suggestion
        # Logs Tensorboard event files as artifacts
```